capabilities/writetarget/beholder/monitor/schema.go

```diff
@@ -0,0 +1,118 @@
+//nolint:gosimple // disable gosimple
+package monitor
+
+import (
+	"fmt"
+	"path"
+	"regexp"
+	"strings"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/runtime/protoimpl"
+)
+
+// TODO: move to another directory / repo
+
+const (
+	AttrKeyBeholderDataSchema = "beholder_data_schema"
```

Comment:
These common attrs can also be published and used from common at one point: https://github.com/search?q=repo%3Asmartcontractkit%2Fchainlink-common%20beholder_data_schema&type=code